pkcs11/src/config/device.rs

```diff
@@ -75,10 +78,6 @@ fn background_timer(
 
         let timeout = next_job_deadline.duration_since(now);
         match rx.recv_timeout(timeout) {
-            Ok(RetryThreadMessage::Finalize) => {
-                jobs.clear();
```

Comment:
I understand we no longer clear the jobs because the thread is cancelled anyway so the jobs are dropped, right?

Reply:
Yes, the background jobs `BTreeMap` is owned by the closure. When a `RecvError` is received it breaks of the main loop and the closure will finish leading to the `BTreeMap` being closed.

Reply:
Previously the sender was in a `static` and thus never dropped.